src/coffee/export.coffee

```diff
@@ -243,12 +242,14 @@ class Export
     if file then fs.writeFileAsync file, content, opts
     else process.stdout.write content
 
-  _parse: (csvString) ->
-    new Promise (resolve, reject) ->
-      Csv().from.string(csvString)
+  _parse: (csvString) =>
+    new Promise (resolve, reject) =>
+      csvString = _.trim(csvString, @options.templateDelimiter)
```

Comment:
But if there is an empty cell **on purpose**? Like no value for a specific attribute?